src/main/java/edu/hm/hafner/analysis/parser/checkstyle/CheckStyleParser.java

```diff
@@ -71,8 +71,8 @@ private Report convert(final CheckStyle collection) {
             if (isValidWarning(file)) {
                 for (Error error : file.getErrors()) {
                     IssueBuilder builder = new IssueBuilder();
-                    mapPriority(error).ifPresent(builder::setSeverity);
 
+                    builder.setSeverity(Severity.guessFromString(error.getSeverity()));
```

Comment:
It is even simpler: `builder.guessSeverity`. I should have noted that in the issue.